.github/workflows/ci-assign-project.yml

```diff
@@ -0,0 +1,59 @@
+name: "CI assign project"
+
+on:
+  pull_request:
+    types: [opened, synchronize, edited, closed]
+
+jobs:
+  cancel-previous-run:
+    name: Cancel previous actions
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: n1hility/cancel-previous-runs@v2
```

Comment:
The recommended way to accomplish this now is via the `concurrency` setting:

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-concurrency-to-cancel-any-in-progress-job-or-run

It seems like the best way to configure concurrency is described here: https://github.com/celestiaorg/celestia-node/pull/894#discussion_r952863065